WordPress/src/main/java/org/wordpress/android/WordPress.kt

```diff
@@ -41,6 +41,8 @@ abstract class WordPress : Application() {
         @JvmField
         var imageLoader: FluxCImageLoader? = null
 
+        fun isWordPressDBInitialized() = ::wpDB.isInitialized
+
```

Comment:
Excellent job! Thank you for discovering this solution. 👏🏻 
I have a few comments:
- I believe the function name should accurately reflect the name of `wpDB`.
- This is a nitpicking, but you could also consider using getter.

WDYT about this change?
```suggestion
        val isWpDBInitialized
            get() = ::wpDB.isInitialized
```

Reply:
Using a property might be more semantically appropriate than using a function actually,